forage-android/src/test/java/com/joinforage/forage/android/pos/ForageTerminalSDKTest.kt

```diff
@@ -78,14 +121,150 @@ class ForageTerminalSDKTest {
     }
 
     @Test
-    fun testCheckBalance() = runTest {
+    fun `POS EBT checkBalance should succeed`() = runTest {
```

Comment:
Potential nit?: These tests are massive. Is it super annoying to break them up somehow? If it is, then let's leave them as is.

Reply:
good callout, reduced duplication + bloating here: e8dc8c3f4b7844dbd76451d21250c0e03c5ba62d